spark/pom.xml

```diff
@@ -731,6 +726,77 @@
       </dependencies>
     </profile>
 
+    <profile>
+      <id>pyspark</id>
```

Comment:
@felixcheung Yes, if you already installed Spark and used SPARK_HOME, you don't need to build Z with this profile.